<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColladaImporter</class>
 <widget class="QDialog" name="ColladaImporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Collada Files:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="m_colladaFilesLabel">
     <property name="toolTip">
      <string>Collada File to Convert</string>
     </property>
     <property name="whatsThis">
      <string>The file you specify here will be send to the Horde3D ColladaConv Tool </string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="m_setColladaFile">
     <property name="whatsThis">
      <string>Let you browse for a collada file</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QCheckBox" name="m_noOptimizeGeometry">
       <property name="toolTip">
        <string>Disable Geometry Optimization in Collada Converter</string>
       </property>
       <property name="whatsThis">
        <string>Disable Geometry Optimization in Collada Converter</string>
       </property>
       <property name="text">
        <string>Do Not Optimize Geometry</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Type:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QComboBox" name="m_type">
       <item>
        <property name="text">
         <string>Model</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Animation</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>LOD distance:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="m_lod">
       <item>
        <property name="text">
         <string>Level 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Level 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Level 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Level 4</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="m_lodDist"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QTextEdit" name="m_logWidget">
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="documentTitle">
         <string>Conversion Log</string>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>221</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_importButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Import converted collada file into repository</string>
       </property>
       <property name="whatsThis">
        <string>Import converted collada file into repository</string>
       </property>
       <property name="text">
        <string>Import</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ColladaImporter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColladaImporter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
